style.css

```diff
@@ -175,4 +179,51 @@ section{
 .quiz-container .answer-wrong{
     background-color: var(--red-color-400);
     color: var(--white-color);
+}
+
+/* || category */
+
+.categories h3{
+    font-size: 2rem;
+    font-weight: 700;
+    margin: 3rem 0;
+    text-align: center;
+}
+
+.category{
+    font-size: 3rem;
+    font-weight: 200;
+    width: fit-content;
+    position: relative;
+    color: var(--text-color);
+}
+
+.category .img-round-sizing{
+    width: 18rem;
+    height: 18rem;
+}
+
+.category-container{
+    display: grid;
+    gap: 1rem;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
```

Comment:
Clean code. Just one suggestion, you could use repeat syntax here for grid, rest nice work! 💯

Reply:
Thanks for the suggestion @nayyyhaa , I will do it.